pkgs/servers/search/sphinxsearch/default.nix

```diff
@@ -13,14 +13,19 @@ stdenv.mkDerivation {
 
   configureFlags = [
     "--program-prefix=sphinxsearch-"
-    "--without-mysql"
+    "--with-mysql=${mariadb}/"
+    "--with-libexpat"
     "--enable-id64"
   ];
 
   nativeBuildInputs = [
     pkgconfig
   ];
 
+  buildInputs = [
+    expat mariadb zlib openssl
```

Comment:
Could you make this one per line? And mention why is zlib or openssl needed?

Reply:
I should reiterate that I'm not an expert. I don't know exactly why those libraries are needed, but without them configure test for MySQL presence will fail as they are linked there after mariadb.